### PR TITLE
Fix january calendar bug with moment > 2.11

### PR DESCRIPTION
**Pull request by ingro**
I think I found the roots of the problem reported on #65.

It seems that if the current month is january the `previousMonthLength` returns `NaN` with moment 2.11+. This happens because the change of year is not factored when calculating the value.

If for example I want to display january 2016, it will try to create a moment in this way to calculate the `previousMonthLength`:
`moment([2016, -1])` which fails due to a stricter handling of invalid dates with moment 2.11+.

So I updated the code and now it should properly create a moment in this way:
`moment([2015, 11])`.

Hope I didn't miss anything!

### CODE REVIEW

File: src/utils/DateUtils.js [hunk 1]
```diff
@@ -20,9 +20,15 @@ const getWeekdaysMin = () => {
 
 const getVisibleDays = (month, year) => {
   const offset = moment([year, month]).startOf('month').weekday();
-  const previousMonthLength = daysInMonthCount(month - 1, year) + 1; //We need the last number too
   const currentMonthLength = daysInMonthCount(month, year) + 1;
 
+  if (month - 1 < 0) {
+    month = 12;
```
Comment (FrancescoCioria) [inline src/utils/DateUtils.js]: why did you change this from `11` to `12`? aren't the months going from `0` to `11`?

Comment (ingro) [inline src/utils/DateUtils.js]: Yep but then it always subtract 1 so the final value is still 11.

Comment (FrancescoCioria) [inline src/utils/DateUtils.js]: yes, I missed that. great!

**Comment by FrancescoCioria** [comment]
thank you @ingro

File: src/utils/DateUtils.js (hunk 1 above)
Comment (ingro) [inline src/utils/DateUtils.js]: I missed that too on the first try :P

**Comment by ingro** [comment]
No problem, always happy to contribute when I can!
